FindTeamer/NetworkReachabilityService/NetworkReachability.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NetworkReachability.swift
+//  FindTeamer
+//
+//  Created by Nikita Sviridovich on 10.10.22.
+//
+
+import SystemConfiguration
+
+final class NetworkReachability {
+    class func isConnectedToNetwork() -> Bool {
```

Comment:
This should be not used as a singletone instance. You won't be able to mock it in tests